Source/WebCore/platform/graphics/gstreamer/mse/WebKitMediaSourceGStreamer.cpp

```diff
@@ -134,10 +134,22 @@ static void enabledAppsrcNeedData(GstAppSrc* appsrc, guint, gpointer userData)
         // Search again for the Stream, just in case it was removed between the previous lock and this one.
         appsrcStream = getStreamByAppsrc(webKitMediaSrc, GST_ELEMENT(appsrc));
 
-        if (appsrcStream && appsrcStream->type != WebCore::Invalid)
-            webKitMediaSrc->priv->notifier->notify(WebKitMediaSrcMainThreadNotification::ReadyForMoreSamples, [webKitMediaSrc, appsrcStream] {
+        if (appsrcStream && appsrcStream->type != WebCore::Invalid) {
```

Comment:
This commit is useful, but is going to be superseeded by a patch @calvaris is working on, so let's better not merge it into wpe-2.28 yet.
See the original discussion about the topic in https://github.com/WebPlatformForEmbedded/WPEWebKit/pull/791

Reply:
@eocanha @calvaris are the changes you're working on will land any time soon? if not, I'd request this change to be merged.

Reply:
I don't mind merging this now. I'll fix what I need to fix later.